src/main/java/com/arpnetworking/http/Routes.java

```diff
@@ -80,16 +80,19 @@
      * @param metricsFactory Instance of <code>MetricsFactory</code>.
      * @param healthCheckPath The path for the health check.
      * @param statusPath The path for the status.
+     * @param supplementalRoutes Optional supplemental routes.
      */
     public Routes(
             final ActorSystem actorSystem,
             final MetricsFactory metricsFactory,
             final String healthCheckPath,
-            final String statusPath) {
+            final String statusPath,
+            final Optional<SupplementalRoutes> supplementalRoutes) {
```

Comment:
How about making this a list?  Having a single SupplementalRoute object doesn't allow for easy composition.